apple/REAModule.mm

```diff
@@ -259,35 +287,57 @@ - (void)sendEventWithName:(NSString *)eventName body:(id)body
 
 RCT_EXPORT_BLOCKING_SYNCHRONOUS_METHOD(installTurboModule : (nonnull NSString *)valueUnpackerCode)
 {
-  facebook::jsi::Runtime *jsiRuntime = [self.bridge respondsToSelector:@selector(runtime)]
-      ? reinterpret_cast<facebook::jsi::Runtime *>(self.bridge.runtime)
-      : nullptr;
-
-  if (jsiRuntime) {
-    auto nativeReanimatedModule = reanimated::createReanimatedModule(
-        self.bridge, self.bridge.jsCallInvoker, std::string([valueUnpackerCode UTF8String]));
-
-    jsi::Runtime &rnRuntime = *jsiRuntime;
-    WorkletRuntimeCollector::install(rnRuntime);
-
 #if __has_include(<UIKit/UIAccessibility.h>)
```

Comment:
can we split that method into two methods? one for bridgeless and one for the rest of cases.

Reply:
Done in [2efade7](https://github.com/software-mansion/react-native-reanimated/pull/5734/commits/2efade7e68c277b2ab8ca61c31814a910dce2703)